sematic/ui/packages/main/src/pipelines/BasicMetricsPanel.tsx

```diff
@@ -0,0 +1,163 @@
+const MetricValue = styled(Typography)`
+  font-size: 70px;
+  font-weight: 500;
+  line-height: 1;
+`;
+
+const MetricLabel = styled(Box)`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  column-gap: ${theme.spacing(1)};
+  justify-content: center;
+
+  & .MuiTypography-root {
+    font-size: 20px;
+  }
+
+  & .MuiSvgIcon-root {
+    color: ${theme.palette.grey[300]};
+  }
+`;
+
+function TopMetric(props: { value: string; label: string; help?: string }) {
+  const { value, label, help } = props;
+
+  return (
+    <MetricBox>
+      <MetricValue sx={{ fontSize: 70, fontWeight: 500, lineHeight: 1 }}>
+        {value}
+      </MetricValue>
+      <MetricLabel>
+        <Typography>{label}</Typography>
+        {help !== undefined && (
+          <Tooltip title={help}>
+            <HelpIcon fontSize="small" />
+          </Tooltip>
+        )}
+      </MetricLabel>
+    </MetricBox>
+  );
+}
+
+export function runSuccessRate(payload: BasicMetricsPayload, run: Run): string {
+  const totalCount = payload.content.total_count;
+  const percentRate =
+    ((payload.content.count_by_state["RESOLVED"] || 0) / (totalCount || 1)) *
+    100;
+  return `${Math.floor(percentRate)}%`;
+}
+
+export function runAvgRunTime(payload: BasicMetricsPayload, run: Run): string {
+  return durationSecondsToString(
+    payload.content.avg_runtime_children[run.calculator_path] || 0
+  );
+}
+
+export default function BasicMetricsPanel() {
+  const { rootRun } = usePipelineRunContext() as { rootRun: Run };
+
+  const [payload, loading, error] = useBasicMetrics({ runId: rootRun!.id });
+
+  const totalCount = useMemo(() => payload?.content.total_count, [payload]);
+
+  const successRate = useMemo(
+    () => (payload ? runSuccessRate(payload, rootRun) : "0%"),
+    [payload, rootRun]
+  );
+
+  const avgRuntime = useMemo(
+    () => (payload ? runAvgRunTime(payload, rootRun) : "0s"),
+    [payload, rootRun]
+  );
+
+  const sortedAvgRuntimeChildren = useMemo(
+    () =>
+      Object.entries(payload?.content.avg_runtime_children || {}).sort((a, b) =>
+        a[1] > b[1] ? -1 : 1
+      ),
+    [payload]
+  );
+
+  return (
+    <Box sx={{ p: 5 }}>
+      {loading === true && <Loading isLoaded={false} />}
+      {!loading && error !== undefined && (
+        <Alert severity="error">Unable to load metrics: {error.message}</Alert>
+      )}
+      {!loading && !error && payload !== undefined && (
+        <>
+          <Typography variant="h1">Pipeline Metrics</Typography>
+          <Grid container sx={{ my: 10 }}>
+            <Grid item xs={4}>
+              <TopMetric
+                value={`${totalCount}`}
+                label="runs"
+                help={`${totalCount} runs of ${rootRun.calculator_path}`}
+              />
+            </Grid>
+            <Grid item xs={4}>
+              <TopMetric
+                value={successRate}
+                label="success rate"
+                help={`Success rate of ${rootRun.calculator_path}`}
```

Comment:
I feel like we should include some info about how it is calculated in the help. Like "Total number of runs that complete successfully divided by total number of pipelines started." Since there are a variety of ways you could define this metric, and people may assume one that's incorrect. Other possible interpretations:

1. Total number of resolved divided by total number in a terminal state
2. Total number of resolved divided by the sum of resolved plus failed (basically remove the CANCELLED runs from (1))
3. `(1 - N_FAILED/TOTAL)` Though possibly intuitive (because people probably *really* care about the failure rate, not the success rate), this would consider actively running or cancelled things to be successes. 

Speaking of, should we consider one of these other alternatives? I am particularly fond of (2), as it makes it so `success rate + failure rate == 1`.

Reply:
That is too much information to include in a tooltip. We can add a page to the docs that details how these metrics are calculated.

I agree we should exclude cancelled runs.